### DISCUSSION TITLE
How to manually specify dimensions for external images?

### DISCUSSION THREAD

**Discussion by TheSenPie** [CaiJimmy/hugo-theme-stack#981] [Q&A]
I usually upload all my images in the project itself, however, I have heavy images, which I want to show with original quality. For that, I use S3 and provide image links in Markdown. The issue is that, in that case, I don't get the zooming feature and the alt-text is not presented. The photos will not change, so I want to hardcode the dimensions some way, to get the same features for external photos. Any clue how I can do that?

**Comment by CaiJimmy** [comment]
> so I want to hardcode the dimensions some way, to get the same features for external photos

Sorry, but that's not supported currently 😓

**Reply by TheSenPie** [reply]
That's unfortunate:/

**Reply by Sciroccogti** [reply]
Then is there a way to rescale internal images?

**Reply by CaiJimmy** [reply]
> Then is there a way to rescale internal images?

Local images (stored in page bundle, not in `public` folder) should have light box effect automatically.

**Reply by Sciroccogti** [reply]
> Local images (stored in page bundle, not in `public` folder) should have light box effect automatically.

Well, I still have another question, is [Image render hooks](https://gohugo.io/render-hooks/images/#default) supported?
